src/main/resources/templates/index.html

```diff
@@ -8,6 +8,6 @@
         <span>React Content Below:</span>
         <div id="content"/>
     </body>
-    <script type="text/javascript" src="http://localhost:3000/app.js">
+    <script type="text/javascript" src="http://localhost:3000/app.jsx">
```

Comment:
This should still be app.js this is generated and the generated code usually gets placed into app.js. This goes along with the comment ion the webpack config.